docs/source/subscriptions.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Adding Subscriptions To Schema
+---
+
+Adding GraphQL subscriptions to your graphQL schema is simple. since Subscriptions are enabled by default in Apollo 2.0 and you dont' need to install anymore any extra packages.
```

Comment:
Something simple like "Apollo Server 2.0 supports GraphQL subscriptions without additional configuration" would be a great lead in

Reply:
fixed